src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1318 @@
+            // but the former is preferred.
+            if ($sHeader == 'Variant/DNA_published') {
+                if (!in_array('VariantOnTranscript/Published_as', $aSections['vot']['db_fields']) &&
+                    in_array('VariantOnGenome/Published_as', $aSections['vog']['db_fields'])) {
+                    // Field available on VOG and not on VOT.
+                    $aOutputHeaders['vog'][$i] = 'VariantOnGenome/Published_as';
+                    continue;
+                } else {
+                    // By default put the published_as field on VOT.
+                    $aOutputHeaders['vot'][$i] = 'VariantOnTranscript/Published_as';
+                    continue;
+                }
+            }
+
+            if (isset($aFieldLinks[$sHeader])) {
+                // Use output header linked in $aFieldLinks.
+                list($sSection, $sHeaderOut) = $aFieldLinks[$sHeader];
+                $aOutputHeaders[$sSection][$i] = $sHeaderOut;
+                continue;
+            }
+
+            // Check if header occurs as a literal DB field.
+            foreach ($aSectionIDs as $sSection) {
+                if (isset($aSections[$sSection]['db_fields'])  &&
+                    in_array($sHeader, $aSections[$sSection]['db_fields'])) {
+                    $aOutputHeaders[$sSection][$i] = $sHeader;
+                    continue 2;
+                }
+            }
+
+            // Try to link custom columns.
+            if (strpos($sHeader, '/') !== false) {
+                list(, $sFieldname) = explode('/', $sHeader, 2);
+                foreach ($aSectionIDs as $sSection) {
+                    $aSection = $aSections[$sSection];
+                    if (isset($aSection['customcol_prefix'])) {
+                        $sNewFieldName = $aSection['customcol_prefix'] . '/' . $sFieldname;
+                        if (in_array($sNewFieldName, $aSection['db_fields'])) {
+                            // Set output header with new LOVD3 prefix (e.g. Individual).
+                            $aOutputHeaders[$sSection][$i] = $sNewFieldName;
+                            continue 2;
+                        }
+                    }
+                }
+
+                // Try to find default custom column translation.
+                foreach ($aCustomColLinks as $sPrefix => $aCustomColDefault) {
+                    if (strpos($sHeader, $sPrefix) === 0) {
+                        list($sSection, $sPrefixOut) = $aCustomColDefault;
+                        $sHeaderOut = str_replace($sPrefix, $sPrefixOut, $sHeader);
+                        $aOutputHeaders[$sSection][$i] = $sHeaderOut;
+                        $_WARNINGS[] = 'Warning: linked "' . $sHeader . '" to non-existing ' .
+                            'column "' . $sHeaderOut . '" in output section "' .
+                            $aSections[$sSection]['output_header'] . '"';
+                        continue 2;
```

Comment:
This warning is triggered all the time for columns that do exist in the database. I can't debug it myself, there's very little comments and I'm not quite following it.